### DISCUSSION TITLE
how  to install EOS

### DISCUSSION THREAD

**Discussion by Markliulei** [cern-eos/eos#44] [General]
dear all. We read the EOS official document about https://eos-docs.web.cern.ch/diopside/index.html. Here want to deploy EOSV5.xxx. Currently, we have 30 VMs  , many problems occur during the deployment. MGM&MQ&QBD services are deployed on three management nodes, MGM&QDB  form a cluster .only FST services are deployed on storage nodes. 
There's a lot of doubt here . Is this deployment architecture correct? How to establish communication between the FST and the management node? where is the configuration file, and how to modify it? In addition, the commands on the official document, which are executed on the management node, which are executed on the real FST node.

**Comment by ayanb07** [comment]
Hi team I asked question related to this in https://github.com/xrootd/xrootd/discussions/2398#discussioncomment-11828394 xrood discussions and found pointers to this repository by seems discussion is not very active. @apeters1971, is this question relevant here?
